crates/kitsune2/tests/blocks.rs

```diff
@@ -292,7 +283,7 @@ pub async fn make_test_peer(builder: Arc<Builder>) -> TestPeer {
     transport.register_module_handler(TEST_SPACE_ID, "test".into(), tx_handler);
 
     // It may take a while until the peer url shows up in the transport stats
-    let peer_url = iter_check!(200, {
+    let peer_url = iter_check!(5000, 100, {
```

Comment:
With the relay iroh endpoints need 3 seconds for creating a net report,after which the home relay is selected. This will be shortened by another modification that bypasses this discovery for tests.